tests/HL_testing_input.json

```diff
@@ -0,0 +1,16 @@
+{
+    "model_name" : ["gpt2","gpt2-medium","TinyLlama/TinyLlama-1.1B-Chat-v1.0","Salesforce/codegen-350M-mono","wtang06/mpt-125m-c4"],
+    "num_cores" : [14],
+    "prompt" : ["My name is"],
+    "prompts_txt_file_path" : ["examples/prompts.txt"],
+    "aic_enable_depth_first" : [0],
+    "mos" : [-1],
+    "cache_dir" : [],
+    "hf_token" : ["NA"],
+    "batch_size" : [1],

```

Comment:
**ERROR!**
It is Failed for batch_size 2 and 4, because batch_size is not evenly divisible by the number of prompts. 
"execute" api is failed.
Note: screenshots is given
![image](https://github.com/quic/efficient-transformers/assets/170617714/8eac2a95-9251-45fd-8401-d9b5f271d026)

Reply:
This is fixed by updating testing script.